bokehjs/src/lib/models/plots/plot_canvas.ts

```diff
@@ -405,6 +406,8 @@ export class PlotView extends LayoutDOMView implements Renderable {
 
   reset_range(): void {
     this.update_range(null)
+    const {x_range, y_range} = this.model
+    this.model.trigger_event(new RangesUpdate(x_range.start, x_range.end, y_range.start, y_range.end))
```

Comment:
This emphasizes why this code should have been factored out into a method on `Plot`.